### PR TITLE
test(AWS Deploy): Add integration tests for custom deployment bucket

### PR DESCRIPTION
**Pull request by AlinoeDoctari**
Closes: #8446

### CODE REVIEW

File: test/integration/custom-deployment-bucket.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  const bucketName = `serverless-test-${uuid.v4()}`;
+
+  before(async () => {
+    await createBucket(bucketName);
+  });
+
+  it('should deploy in the configured aws bucket', async () => {
+    // we cannot deploy an empty fixture like aws so we go for a small one
+    const { servicePath } = await fixtures.setup('function', {
```
Comment (pgrzesik) [inline test/integration/custom-deployment-bucket.test.js:21]: Let's move `setup/teardown` of fixture to `before/after` hooks similar to other integration tests

**Review by pgrzesik — CHANGES_REQUESTED** [review]
Thank you @AlinoeDoctari :bow: It looks great, I just have one suggestion of moving deploy and remove to `before` and `after` hooks

File: test/integration/custom-deployment-bucket.test.js (hunk 1 above)
Comment (AlinoeDoctari) [reply test/integration/custom-deployment-bucket.test.js:21]: Should that include the `deployService/removeService`?

Comment (pgrzesik) [reply test/integration/custom-deployment-bucket.test.js:21]: Yes, sorry for not being specific here :+1:

**Review by pgrzesik — APPROVED** [review]
Awesome job @AlinoeDoctari, thank you :bow:

**Pull request merged**